### ISSUE TITLE
1/6/24-hour average hashrate stats problem!

### ISSUE THREAD

**Issue by zairoo** [dvandal/cryptonote-nodejs-pool#314]
after upgrade, redis unable to create 
***coin:charts:hashrate:wallet 
***coin:charts:worker_hashrate:wallet~workname

restarted the pool Waiting for a long time and not creating it

**Comment by travis1234** [comment]
maybe check if your config has worker_hashrate under user in charts.

```
            "worker_hashrate": {
                "enabled": true,
                "updateInterval": 60,
                "stepInterval": 60,
                "maximumPeriod": 86400
            },
```

**Comment by zairoo** [comment]
Yes, I am using this, but upgrading from old redis data is not available. But emptying the redis database, there is no problem

**Comment by travis1234** [comment]
weird can you create those manually in redis?

**Comment by zairoo** [comment]
version 1.3.5 is the database problem?

**Comment by muscleman** [comment]
it might be an authentication mis match in api.js function authorize. when local calls are made no password is used only the remoteaddress to determine internal or external calls.
try setting the bindIp in config.json to your ip
